static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -147,6 +148,8 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
   }>(jsFrameworkSelectOptions[0]);
 
   const defaultTab = 'npm';
+  const {location} = useRouteContext();
+  const crashReportOnboarding = location.hash === '#crashreport-sidequest';
```

Comment:
this hash `#crashreport-sidequest` is how we tell we're coming from the issue details tab